app/views/seances/_seance.html.erb

```diff
@@ -1,27 +1,12 @@
 <div id="<%= dom_id seance %>">
   <p>
     <strong>Start time:</strong>
-    <%= seance.start_time %>
-  </p>
-
-  <p>
-    <strong>End time:</strong>
-    <%= seance.end_time %>
+    <%= seance.start_time.strftime('%d.%m %H:%M') %>
```

Comment:
If this is a standard time formatting we will use on the whole page it should be extracted to a helper method so that we don't get mixed up and also if we need to change it we could change it in one place
I think I saw such method being added already in another PR,
It would be a good idea to ask your team about it and make sure everyone uses the same formatting if you didn't discuss that already

Reply:
Moved the formatting to a helper, thanks.